qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -567,3 +567,37 @@ def test_abort_conn(self):
         self.assert_session_state(gid, "open")
         time.sleep(session_timeout * 1.5)  # Long enough for MDS to consider session stale
         self.assert_session_state(gid, "stale")
+
+    def test_dont_mark_unresponsive_client_stale(self):
+        """
+        Test that an unresponsive client holding caps not required by any
+        other client is evicted directly, without being marked as stale even
+        after session_timeout, at session_autoclose.
+        """
+        if not isinstance(self.mount_a, FuseMount):
+            self.skipTest("Require FUSE client to handle signal STOP/CONT")
+
+        # XXX: To conduct this test we need at least two clients since a
+        # single client is never evcited by MDS.
+        SESSION_TIMEOUT = 30
+        SESSION_AUTOCLOSE = 50
+        mount_a_gid = self.mount_a.get_global_id()
+        mount_a_pid = self.mount_a.get_client_pid()
```

Comment:
You're not using this variable anymore. Please remove.

Reply:
I should've gotten rid of it.